README.md

```diff
@@ -1,2 +1,115 @@
-# hello-rails-react
-This is a simple hello rails app that is created in reils and react with webpack.
+# Hello-Rails-React
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Hello Rails React <a name="about-project"></a>
+
+**Hello Rails React** is a project which using Ruby on Rails for back and and React for front end in one project. You can generate greeting messages randomly.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+
+### Key Features <a name="key-features"></a>
+
+-Creating API end point by Rails
+-Using API data and display in React 
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby
+- Rails
+- React
+
+### SetUp
+
+Clone this Repository to your desired folder:
+
+``` sh
+cd my-folder
+git clone https://github.com/otmanTR/hello-rails-react-.git
+```
```

Comment:
- [ ] Kindly pay more intention to the information you give in your README file, please kindly fix this command link 🙃